server/agent/tools.py

```
@@ -0,0 +1,172 @@
+    """Create a document file in /workspace/output/. Supports .md, .txt formats.
+
+    For .docx or .xlsx files, use create_docx or create_xlsx instead.
+    """
+    output_dir = "/workspace/output"
+    try:
+        os.makedirs(output_dir, exist_ok=True)
+        filepath = os.path.join(output_dir, filename)
+        with open(filepath, "w") as f:
+            f.write(content)
+        return {"filename": filename, "path": filepath, "status": "created"}
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)